src/__tests__/config/farms.test.ts

```diff
@@ -36,17 +35,6 @@ describe('Config farms', () => {
     ).toBeTruthy()
   })
 
-  it.each(farmsToTest)('Farm %d has non 0 tokens amount', async (pid, farm) => {
```

Comment:
PRs for farms are always submitted before we add the funds, so this check is always ignored. Better remove it.